packages/create-plugin/src/utils/utils.templates.ts

```diff
@@ -88,23 +92,70 @@ export function renderTemplateFromFile(templateFile: string, data?: any) {
   return renderHandlebarsTemplate(fs.readFileSync(templateFile).toString(), data);
 }
 
-export function getTemplateData(): TemplateData {
+export function buildProxyFromJson() {
   const pluginJson = getPluginJson();
+  return new Proxy(pluginJson, {
```

Comment:
May I ask what the benefits are of using a proxy here are?

Reply:
(Resolving this, as the proxy got removed.)